runtime/doc/options.txt

```diff
@@ -5159,8 +5159,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 	unescaping, so to keep yourself sane use |:let-&| like shown above.
 							*shell-powershell*
 	To use powershell (on Windows): >
-		set shell=powershell shellquote=( shellpipe=\| shellredir=> shellxquote=
+		set shell=powershell shellquote=( shellpipe=\| shellxquote=
```

Comment:
I wonder what `>`  is equivalent to exactly.

Reply:
utf-16 on Windows.
utf-8 on Unix.